lib/takeaway.rb

```diff
@@ -0,0 +1,50 @@
+
+class Takeaway
+  require_relative './menu.rb'
+  require_relative './order.rb'
+  require_relative './sms_sender.rb'
+
+  attr_reader :menu, :order
+  
```

Comment:
Something doesn't feel quite right about the idea of initializing a new Order for each new Takeaway...  Do you need to create a new Takeaway for each order?  I suspect the answer to that question is in your code somewhere, and I'll find it soon!

Reply:
Otherwise, it's good that you're injecting the dependencies here 👍